ethereum/README.md

```diff
@@ -28,7 +28,11 @@ For each test run:
 
 Run `npx ganache-cli --deterministic --time "1970-01-01T00:00:00+00:00"` to start a chain.
 
-Run the tests using `npm run test`
+Run the all ethereum tests using `npx truffle test`
```

Comment:
Can we include the
```
make test
```
route as an alternative, which takes care of spinning up ganache and running the tests

Reply:
yep, 💯